packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -183,19 +183,18 @@ export function createOdspNetworkError(
 
     error.online = OnlineStatus[isOnline()];
 
-    const errorAsAny = error as any;
-
-    errorAsAny.response = responseText;
+    const props: ITelemetryProperties = { response: responseText };
```

Comment:
Ok I'm having a major double-take here - I did this _exact_ change in a branch at one point but dropped it...  Great minds think alike I guess!